src/content/contributing/designers/designers.mdx

```diff
@@ -1,30 +1,58 @@
-- [Heavy](https://github.com/carbon-design-system/carbon-contribution/wiki/3.-Heavy-Contribution-Model): Contributing a whole new component to the system
+</AnchorLinks>
 
-### Submitting an issue
+## How to help
 
-If you are not sure which model your contribution fits into, open an issue in one our repos and we will figure out which model it falls into.
+### Leave feedback
 
-#### Not sure where to open an issue?
+The best place to leave feedback is [GitHub](https://github.com/carbon-design-system/carbon/issues/new/choose). If you're not already a member, it takes just a minute to [join](https://github.com/join). Carbon offers a few feedback templates, which make sure an issue includes all the information necessary for the team to take a look.
 
-All contribution proposals should be opened in the [carbon-contribution repo](https://github.com/carbon-design-system/carbon-contribution). This includes light, medium, and heavy contributions (all components, icons, guideline updates, etc.).
+### Fix a bug
 
-#### Not sure how to submit an issue?
+If you find something that's broken, you can let us know with [an issue](https://github.com/carbon-design-system/carbon/issues/new/choose). If you know how to fix it, and want to make Carbon even better, you can fix the bug and submit a pull request!
 
-It's not so bad, we promise! You can read a [tutorial](https://help.github.com/articles/creating-an-issue/) on GitHub to get familiar with best practices.
+1. Find a bug
+2. Document the bug with a GitHub issue
+3. Fix the bug and submit a pull request
 
-## Before you start
+If you're interested in contributing back but don't know of any bugs, you can tackle any of the open issues in [our repo](https://github.com/carbon-design-system/carbon/issues).
+
+### Add a feature
+
+If Carbon already has a component or pattern you need, but it's missing a feature you need, you're encouraged to expand the asset and contribute back.
+
+1. Design the component or pattern with the feature added
+2. Create a specification a developer can act on
+3. Build the expanded component in at least one of Carbon's supported frameworks
+4. Extend the existing  Sketch symbol
+5. Document how to use the component’s new feature
+
+Feature additions make Carbon better for everybody. If your team needs something, other teams probably do too.
+
+### Create something new
+
+Before a component, pattern, or tool can become a core Carbon asset, each of these steps must be completed (but not necessarily by you).
+
+1. Design the component or pattern
+2. Create a specification a developer can act on
+3. Build the component in at least one of Carbon's supported frameworks
+4. Create a Sketch symbol for the component
+5. Document how to use the component or pattern
+
+Steps 1 and 2 must be completed to contribute. All contributions are valuable. If you have a design spec, but lack the time or resources to build it, the community may pick it up.
```

Comment:
we _need_ a feature enhancement issue; contributors can create any, some, or all of the items 1, 2, 3, 4 5, 6; for components, we need all 6 to be considered for main branch inclusion.